src/csexp_rpc/dune

```diff
@@ -6,7 +6,6 @@
   dyn
   dune_util
   csexp
-  dune_engine
```

Comment:
How was this dependency used? I don't see any deleted code, so it's all rather mysterious.

Reply:
Ah, sorry, it's `include Dune_engine.Scheduler.Worker`! Got it now.